nncf/onnx/graph/metatypes/onnx_metatypes.py

```diff
@@ -85,6 +86,22 @@ def matches(cls, model: onnx.ModelProto, node: onnx.NodeProto) -> bool:
         return _is_depthwise_conv(model, node)
 
 
+@ONNX_OPERATION_METATYPES.register()
+class ONNXGroupConvolutionMetatype(ONNXOpWithWeightsMetatype):
```

Comment:
Should be added groupconv metatypes in PT backend?

Reply:
If it is possible, we may extend metatypes in the PT backend. But not in this PR, I hope.

Reply:
135885